### PR TITLE
add: `rustdesk-deb`

### PR DESCRIPTION
**Pull request by Xdavius**
PacReq: rustdesk #5644

### CODE REVIEW

**Comment by Xdavius** [comment]
@Elsie19 please, can you review ? it's my first add. thank you

**Comment by Elsie19** [comment]
Fix the title.

**Comment by Xdavius** [comment]
> Fix the title.

Done

File: packages/rustdesk-deb/rustdesk-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name="rustdesk-deb"
+gives="rustdesk"
+repology=("project: ${gives}")
+pkgver="1.2.3"
+url="https://github.com/rustdesk/rustdesk/releases/download/${pkgver}-1/rustdesk-${pkgver}-x86_64.deb"
+breaks=("${gives}-bin")
+pkgdesc="Yet another remote desktop software, written in Rust."
```
Comment (Elsie19) [inline packages/rustdesk-deb/rustdesk-deb.pacscript:7]:
```suggestion
pkgdesc="Yet another remote desktop software, written in Rust"
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/rustdesk-deb/rustdesk-deb.pacscript (hunk 1 above)
Comment (Xdavius) [reply packages/rustdesk-deb/rustdesk-deb.pacscript]: Done

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**